prime-router/docs/api/README.md

```diff
@@ -1,16 +1,79 @@
+# Swagger UI
+
+Swagger UI is a tool that allows you to visually interact with ReportStream APIs that have been defined using the OpenAPI
+Specification (OAS). It provides a user-friendly interface that makes it easier to understand and explore capabilities of
+the API without requiring external tools such as curl or Postman.
+
+This directory contains all open api spec resources:
 
 1. ./: specs in yaml format derived from other process (vs auto generated specs under ./generated)
 2. ./swagger-ui: swagger ui artifacts (with customization - see custom and maintain notes below)
 3. ./generated: auto generated api spec(s) from annotated kotlin source files
 
-swagger ui customization and maintainance:
+## Purpose
 
-swagger ui dist update history:
+The main purpose of Swagger UI is to:
 
-|    date    |  dist |              comments                |
-|------------|-------|--------------------------------------------------------------|
-| 07/24/2023 | 5.1.0 | only change made in swagger-initializer.js to prevent js injection |
+1. **Documentation:** generates documentation in a terse yet approachable format that serves as great reference
+   material.
+
+2. **API Visualization:** provides a structured interface that displays API endpoints, request parameters, response 
+   structures, and example requests and responses in a structured, readable, and navigable format.
+
+3. **API Testing:** facilitates direct interaction with the API. You can input parameters, make requests, and
+   see the responses right in your browser with no external tools.
+
+## Usage
+With an API selected the UI will display a list of endpoints. Each endpoint can be expanded to display the
+usage, parameters, and expected output; as well as an interface to make an API call.
+
+### Accessing the Swagger UI Interface
+
+The interface consists of static web assets that are compiled and served similarly to our front-end web application.
+The UI is served and accessible on the same server as the reportstream web ui / front-end using a particular URL / path.
+To access it, navigate to TLD for your chosen environment and append `/swagger-ui` to the end of the root url.
+
+**Staging** https://staging.prime.cdc.gov/swagger-ui
+**Local** https://localhost:8080/swagger-ui
+
+Please note that while all environments can be selected in the swagger-ui (regardless of your chosen URL),
+you may encounter cross-origin restrictions when trying to access one environment from outside its URL.
+
+### Authentication
 
+Swagger has built-in support for authentication; it will map user-provided credentials into the appropriate headers and
+even handle obtaining and refreshing a bearer token using OAuth.
 
+Regardless of chosen authentication method, the steps are similar:
+Near the top-right of the interface you'll find an `Authorize` button which will bring up a menu with the authentication
+methods available (described below). Each option will require some inputs.
 
+#### OAuth
```

Comment:
I would call this Okta instead of OAuth? Unless it works for other OAuth cases? If so, then document somewhere that presently this is used for Okta auth